subsys/net/lib/openthread/platform/uart.c

```diff
@@ -29,25 +29,26 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #include "platform-zephyr.h"
 
 struct openthread_uart {
-	struct ring_buf *tx_ringbuf;
 	struct ring_buf *rx_ringbuf;
 	struct device *dev;
 	atomic_t tx_busy;
 	atomic_t tx_finished;
 };
 
 #define OT_UART_DEFINE(_name, _ringbuf_size) \
-	RING_BUF_DECLARE(_name##_tx_ringbuf, _ringbuf_size); \
 	RING_BUF_DECLARE(_name##_rx_ringbuf, _ringbuf_size); \
 	static struct openthread_uart _name = { \
-		.tx_ringbuf = &_name##_tx_ringbuf, \
 		.rx_ringbuf = &_name##_rx_ringbuf, \
 	}
 
 OT_UART_DEFINE(ot_uart, CONFIG_OPENTHREAD_NCP_UART_RING_BUFFER_SIZE);
 
 #define RX_FIFO_SIZE 128
 
+static bool is_panic_mode;
+static const uint8_t *s_write_buffer;
+static uint16_t       s_write_length;
```

Comment:
I'd unify the approach, whether you align the variables or not. Generally, it's not needed.

Reply:
Done